src/lib/Board.ts

```diff
@@ -0,0 +1,108 @@
+import Instance from './Instance';
+import Coordinate from './Coordinate';
+import Cell from './Cell';
+import { clone, addItem, removeItem } from './ArrayEdit';
+
+const BOARD_SIZE = 9;
+export default class SudokuBoard extends Instance {
+  private _board: Cell[];
+  private _displayCache: string | null;
+
+  constructor() {
+    super();
+    this._board = [];
+    this._displayCache = null;
```

Comment:
```suggestion
    delete _displayCache;
```
Removing the key from memory entirely may be better here